features/handle-mention.feature

```diff
@@ -0,0 +1,7 @@
+Feature: Incoming mentions
+
+    Scenario: We receive a Create(Note) with a mention
+        Given an Actor "Person(Alice)"
+        And a "Create(Note)" Activity "Note" by "Alice" with content "Hello @index@site.com" that mentions "Us"
+        When "Alice" sends "Note" to the Inbox
+        Then the request is accepted
```

Comment:
I wanted to do:

Then `a notification of type "Mention" is created`, in which I request the notifications endpoint and check that a mention-type notification is present - but couldn't get that to work. Skipping it for now for the sake of getting the rest of the code in, I'll try to get that working in another PR